internal/monitors/traefik/metadata.yaml

```diff
@@ -0,0 +1,297 @@
+monitors:
+- monitorType: traefik
+  doc: |
+    ### DESCRIPTION
+
+    Configure <a target="_blank" href="https://docs.traefik.io/configuration/metrics">Traefik</a> to expose internal
```

Comment:
The description starts with an imperative.  This should be something more descriptive like "Traefik is a ..."